### PR TITLE
MyReports - Front-end foundation. (Wizard to single page)

### PR DESCRIPTION
**Pull request by galitskyd**
Added an initial page:
screen shot 2015-03-06 at 1 45 47 pm

Fields in a report generate without a need of a template and no more wizard:
screen shot 2015-03-06 at 1 46 38 pm

Expandable/Collapsable lists:
screen shot 2015-03-06 at 1 48 07 pm

with scrolling:
screen shot 2015-03-06 at 1 48 17 pm

A bad looking review modal before a User generates a report:
screen shot 2015-03-06 at 1 50 43 pm

### CODE REVIEW

File: static/my.jobs.161-18.css [hunk 1]
```diff
@@ -1363,6 +1363,121 @@ ul.header-menu > li > ul > a > li:hover {
...
+.list-body li:hover {
+    background-color: #D5FACF;
+    cursor: pointer;
+}
+
+.list-body ul input[type="checkbox"] {
+    margin-top: -1px;
+}
+
+.invis-box {
```
Comment (aspidites) [inline static/my.jobs.161-18.css]: That invis-box, though...

File: static/reporting.162-06.js [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+  for (key in data) {
+    if (data.hasOwnProperty(key)) {
+      var value = data[key];
+
+      // Replace _ with spaces
+      key = key.replace(/_/g, " ");
+
+      html += "<label>" + key + ":</label>";
+
+      // If value is an object (aka a list).
```
Comment (aspidites) [inline static/reporting.162-06.js]: do we need this comment?

**Comment by aspidites** [comment]
Played with it locally, and everything seems to work on firefox now. Do we want to make sure it works on IE 8 before merging?

File: static/my.jobs.161-18.css (hunk 1 above)
Comment (aspidites) [inline static/my.jobs.161-18.css]: Pretty sure we don't need .invis-box anymore.

**Comment by aspidites** [comment]
Are the places where `!=` (instead of `!==`) are used deliberate?

**Comment by galitskyd** [comment]
There aren't any '!=' in the new code.

**Comment by aspidites** [comment]
Bleh. You're right. I was looking at the wrong file.

**Pull request merged**